governance.md

```diff
@@ -0,0 +1,57 @@
+
+## Governance
+
+This section describes the governance and leadership model of The Project.
+
+The foundations of Project governance are:
+
+- Openness & Transparency
+- Active Contribution
+- Institutional Neutrality
+
+The Project leadership consists of a BDFL (Arfon Smith) and Steering Council.
+
+### BDFL
+
+The Project will have a BDFL (Benevolent Dictator for Life), who is currently Arfon Smith. As Dictator, the BDFL has the authority to make all final decisions for The Project. The BDFL, being benevolent, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL is chair of the Steering Council (see below) and may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.
+
+The BDFL can appoint his/her successor, but it is expected that the Steering Council would be consulted on this decision. If the BDFL is unable to appoint a successor, the Steering Council will make a suggestion or suggestions to the main NumFOCUS Board. While the Steering Council and main NumFOCUS Board will work together closely in selecting a BDFL, the main NumFOCUS Board will make the final decision.
+
+### Steering Council
+
+The Project will have a Steering Council that is elected to further the overall mission of the The Project. The initial Steering Council was created by the BDFL, however, future membership rules will determined by the Steering Council itself. The overall role of the Steering Council is to ensure, through working with the BDFL and taking input from the community, the long-term well-being of the project, both technically, editorially, and as a community.
+
+The Steering Council and its Members play a special role in certain situations. In particular, the Council may:
+
+- Make decisions about the overall scope, vision, and direction of The Project.
+- Make decisions about strategic collaborations with other organizations or individuals.
+- Make decisions about new starting a new journal (or incorporating an existing journal) under the Open Journals organization.
+- Make decisions about the Services that are run by The Project and manage those Services for the benefit of the Project and Community.
+- Make decisions when regular community discussion doesn’t produce consensus on an issue in a reasonable time frame.
+
```

Comment:
Likely should add in something about financial oversight and planning. This would include being able to apply for grants too. 
- Provide financial oversight and planning for the The Project

Reply:
I see below this has been separated out into another subcommittee, so just wondering where it fits best.